src/ft2font.cpp

```diff
@@ -360,6 +371,13 @@ FT2Font::FT2Font(FT_Open_Args &open_args, long hinting_factor_) : image(), face(
 
     FT_Matrix transform = { 65536 / hinting_factor, 0, 0, 65536 };
     FT_Set_Transform(face, &transform, 0);
+
+    // Set fallbacks
+    for (FT2Font* f : fallback_list)
+    {
+        fallbacks.push_back(f);
+
+    }
```

Comment:
Can we use std::copy instead of the raw loop here?

Reply:
```suggestion
    std::copy(fallback_list.begin(), fallback_list.end(), std::back_inserter(fallbacks));
```

Reply:
I think we can allow C++11 at least (maybe even 14), so I think that's fine?